src/librustc/hir/lowering.rs

```diff
@@ -2459,12 +2459,12 @@ impl<'a> LoweringContext<'a> {
         }
     }
 
-    fn lower_arg(&mut self, arg: &Arg) -> hir::Arg {
+    fn lower_param(&mut self, param: &Param) -> hir::Arg {
```

Comment:
We should consistently use `Param` in HIR as well.

Reply:
Ok, changed in Commit #e64fccf